src/SprykerEco/Zed/Computop/Dependency/Facade/ComputopToCalculationInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Dependency\Facade;
+
+use Generated\Shared\Transfer\OrderTransfer;
+
+interface ComputopToCalculationInterface
```

Comment:
As per new RFC this could be layer included: `ComputopToCalculationFacadeInterface`